test/unit/specs/components/Personalization/createExecuteDecisions.spec.js

```diff
@@ -143,4 +143,28 @@ describe("Personalization::createExecuteDecisions", () => {
       expect(logger.error).toHaveBeenCalledWith(error);
     });
   });
+
+  it("should not trigger collect when dom-action click", () => {
+    executeActions = jasmine
+      .createSpy()
+      .and.returnValues([undefined], [undefined]);
+    const spy = jasmine.createSpy();
```

Comment:
I noticed these module spies (in this and other tests) aren't actually used for spying and maybe not used for anything at all. Maybe do this instead if you need `foo` to be a function:

```
const modules = {
  foo() {}
}
```

Or if you do need foo to be a spy for one of the tests:

```
const modules = jasmine.createSpyObj("modules", ["foo"]);
```

Reply:
yes, I'll refactor these tests. Thank you